frontend/src/views/toolbox/clam/index.vue

```diff
@@ -146,7 +146,7 @@
                         </template>
                     </el-table-column>
                     <fu-table-operations
-                        width="200px"
+                        width="300px"
                         :buttons="buttons"
                         :ellipsis="10"
                         :label="$t('commons.table.operate')"
```

Comment:
The only change is that the `width` property has been changed from "200px" to "300px". This should have no impact on functionality but might visually affect the table's layout.